.travis.yml

```diff
@@ -5,12 +5,10 @@ before_install:
     # disable rvm, use system Ruby
     - rvm reset
     - wget https://raw.githubusercontent.com/yast/yast-devtools/master/travis-tools/travis_setup.sh
-    - sh ./travis_setup.sh -p "rake yast2-devtools yast2-testsuite yast2 yast2-inetd ruby-rspec" -g "yast-rake gettext"
+    - sh ./travis_setup.sh -p "rake yast2-devtools yast2 ruby-rspec" -g "yast-rake gettext"
 script:
     - rake check:syntax
     - rake check:pot
-    - make -f Makefile.cvs
-    - make
-    - sudo make install
-    - make check
+    - sudo rake install
+    - rake test:unit check
```

Comment:
NP: the usual workflow is to test first and then install, but if that requires some `$LOAD_PATH` or `$Y2DIR` tweaks I'm fine with that. (And in Travis it does not matter much.)